config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -68,14 +68,14 @@ spec:
                 pytorch:
                   properties:
                     modelClassName:
-                      description: Name of the model class for PyTorch model
                       type: string
                     modelUri:
                       type: string
                     resources:
-                      description: Defaults to requests and limits of 1CPU, 2Gb MEM.
+                      description: Defaults to latest Runtime Version.
```

Comment:
Why the description changed here?

Reply:
For some reason codegen didnt run on #153? if you look at the code, this is what it should be:
```
	// Defaults to latest Runtime Version.
	Resources v1.ResourceRequirements `json:"resources,omitempty"`
```

Reply:
oh it is wrong comments then